modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/cmd/impl/create/CreateExchangeCmd.java

```diff
@@ -70,13 +70,11 @@ public void execute() {
                 .sendHttpRequest(new HttpRequest(urlSuffix, exchange.getAsJsonString()), "POST");
 
         // handle response
-        try {
-            JSONParser jsonParser = new JSONParser(JSONParser.MODE_PERMISSIVE);
-            AbstractCmd.OUT_STREAM.println(jsonParser.parse(response.getPayload()).toString());
-        } catch (ParseException e) {
-            BrokerClientException parseException = new BrokerClientException();
-            parseException.addMessage("error while parsing broker response for exchange creation" + e.getMessage());
-            throw parseException;
+        if (response.getStatusCode() == 201) {
```

Comment:
Can we use `HttpURLConnection.HTTP_CREATED` here instead of 201?

Reply:
+1, thanks for pointing it out :)